lib/diaspora_federation/parsers/json_parser.rb

```diff
@@ -0,0 +1,60 @@
+module DiasporaFederation
+  module Parsers
+    # This is a parser of JSON serialized object. JSON object format is defined by
+    # JSON schema which is available at [...].
+    # TODO: We must publish the schema at a real URL
```

Comment:
Our schema URL contains a version number: `https://diaspora.github.io/diaspora_federation/schemas/federation_entities-0.2.0-dev.json`. If we write it here with the version number, then we'll have to update the link everytime we update the version number. Maybe we should make a symlink to the latest instead?